src/nav/navActions.js

```diff
@@ -33,7 +33,7 @@ export const resetToMainTabs = (): GenericNavigationAction =>
 
 /** Only call this via `doNarrow`.  See there for details. */
 export const navigateToChat = (narrow: Narrow): GenericNavigationAction =>
-  StackActions.push('chat', { narrow });
+  StackActions.push('chat', { narrow, editMessage: null });
```

Comment:
`editMessage: null` could go here, or it could go in `AppNavigator` where we define the screen ([doc](https://reactnavigation.org/docs/params/#updating-params)):

```
<Stack.Screen name="chat" component={ChatScreen} initialParams={{ editMessage: null }} />
```